TWLight/users/forms.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, *args, **kwargs):
         self.helper = FormHelper()
         self.helper.add_input(
             Submit(
-                "submit",
+                "Submit",
```

Comment:
Don't think this does what you probably think it does. The first argument in `layout.Submit()` is the `name` of the submit button, which is used to reference the correct button on POST and is not user facing. That would be the next argument (`"Update profile"` in this case), which forms the `value` of the button, which _is_ user facing. And by convention, names, ids, classes etc are typically in lowercase, to improve readability and to avoid confusion.